code/modules/shuttle/shuttle.dm

```diff
@@ -735,14 +735,12 @@ GLOBAL_LIST_INIT(shuttle_turf_blacklist, typecacheof(list(
 		shuttle_area.parallax_movedir = FALSE
 	if(assigned_transit && assigned_transit.assigned_area)
 		assigned_transit.assigned_area.parallax_movedir = FALSE
-	var/list/L0 = return_ordered_turfs(x, y, z, dir)
-	for (var/thing in L0)
-		var/turf/T = thing
-		if(!T || !istype(T.loc, area_type))
+	for (var/client/C in SSmobs.clients_by_zlevel[z])
```

Comment:
```suggestion
	for (var/client/C as() in SSmobs.clients_by_zlevel[z])
```

Reply:
This usage is wrong, SSmobs.clients_by_zlevel contains mobs and not clients. Will need to look into this.